controllers/operator.go

```diff
@@ -410,6 +431,7 @@ func (o *operator) constructDeploy(partition int32) *appsv1.Deployment {
 	}
 	deploy.Annotations[PartitionAnnotation] = strconv.Itoa(int(partition))
 	deploy.Annotations[GenerationAnnotation] = o.observedGeneration()
+	deploy.Spec.Template.ObjectMeta.Labels = map[string]string{KonsumeratorManagerAnnotation: "true"}
```

Comment:
1. why your constant is called ~Annotations if you set it into label?
2. Please set this label to the `deployLabels` above as well

Reply:
Yes, it should be a label under `ObjectMeta` not an annotations. Thanks for pointing out. Should be fixed now.